scripts/python/getdeps2.py

```diff
@@ -19,232 +19,302 @@
 import os.path
```

Comment:
Please submit changes to getdeps2.py as a separate pull request.  
Some of this will need to be reworked as engr_mode.py is being dropped.